src/pages/groupComparison/GroupComparisonMutationTable.tsx

```diff
@@ -0,0 +1,347 @@
+                        mergeOncoKbIcons: this.props.mergeOncoKbIcons,
+                        oncoKbContentPadding: calculateOncoKbContentPadding(
+                            this.oncokbWidth
+                        ),
+                        pubMedCache: this.props.pubMedCache,
+                        civicGenes: this.props.civicGenes,
+                        civicVariants: this.props.civicVariants,
+                        enableCivic: this.props.enableCivic as boolean,
+                        enableOncoKb: this.props.enableOncoKb as boolean,
+                        enableMyCancerGenome: this.props
+                            .enableMyCancerGenome as boolean,
+                        enableHotspot: this.props.enableHotspot as boolean,
+                        enableRevue:
+                            !!this.props.enableRevue && this.shouldShowRevue,
+                        userDisplayName: this.props.userDisplayName,
+                        indexedVariantAnnotations: this.props
+                            .indexedVariantAnnotations,
+                        resolveTumorType: haveDifferentCancerTypes(
+                            d,
+                            this.resolveTumorType
+                        )
+                            ? () => 'Cancer of Unknown Primary'
+                            : this.resolveTumorType,
+                    },
+                    haveDifferentCancerTypes(d, this.resolveTumorType)
+                )}
+            </span>
+        );
+
+        // generate namespace columns
+        const namespaceColumns = createMutationNamespaceColumns(
+            this.props.namespaceColumns
+        );
+        _.forIn(
+            namespaceColumns,
+            (column: MutationTableColumn, columnName: string) => {
+                this._columns[columnName] = column;
+            }
+        );
+
+        // disable annotation column if non canonical transcript is selected
+        this._columns[MutationTableColumnType.ANNOTATION].shouldExclude = () =>
+            this.props.isCanonicalTranscript === false;
+
+        // order columns
+        this._columns[MutationTableColumnType.PROTEIN_CHANGE].order = 10;
+        this._columns[MutationTableColumnType.ANNOTATION].order = 20;
+        this._columns[MutationTableColumnType.MUTATION_TYPE].order = 30;
+        this._columns[MutationTableColumnType.NUM_MUTATED_GROUP_A].order = 40;
+        this._columns[MutationTableColumnType.NUM_MUTATED_GROUP_B].order = 50;
+        this._columns[MutationTableColumnType.MUTATION_OVERLAP].order = 60;
+        this._columns[MutationTableColumnType.LOG_RATIO].order = 70;
+        this._columns[MutationTableColumnType.P_VALUE].order = 80;
+        this._columns[MutationTableColumnType.Q_VALUE].order = 90;
+        this._columns[MutationTableColumnType.ENRICHED_IN].order = 100;
+        this._columns[MutationTableColumnType.MUTATION_STATUS].order = 110;
+
+        //Adjust column visibility according to portal.properties
```

Comment:
Ideally we shouldn't be mentioning about `portal.properties` in the frontend codebase. But seems like we already did the same thing in `ResultsViewMutationTable` and `PatientViewMutationTable`. It's also a bit concerning that we keep duplicating certain segments of code in each new Mutation Table implementation. I guess this is also something we need to address later.